src/farmer.py

```diff
@@ -153,9 +152,10 @@ async def connection_check():
                     if full_node_retry:
                         log.info(f"Reconnecting to full_node {full_node_peer}")
                         if not await self.server.start_client(
-                            full_node_peer, None, self.config
```

Comment:
@Yostra Is it correct to use auth=True for farmer, timelord, and harvester connections?

Reply:
Right now only farmer's server enforces authentication for incoming connections, so auth=True should be used only for connection towards the farmer.